t/re/reg_mesg.t

```diff
@@ -20,6 +20,36 @@ use open qw(:utf8 :std);
 use warnings;
 BEGIN { ${^WARNING_BITS} = undef }  # Kludge to restore default warnings
 
+# Escape a string in a similar fashion to how the regex debugger does,
+# using "x" style escapes in the form "%x{..}" to show the codepoint of
+# the escaped value and "%%" to show a literal percentage sign.  We use
+# percentage instead of backslash so that it is trivial to distinguish
+# backslash based sequences that are in the pattern from escaped octets.
+# Currently we assume ASCII/Unicode semantics, and escape anything that
```

Comment:
After the change below, this is no longer true.  I suggest something like

> We escape anything that isn't an ASCII printable character

The 32 and 126 are less important

Reply:
Comment has been updated.